src/components/DayPicker.jsx

```diff
@@ -263,6 +265,12 @@ class DayPicker extends React.PureComponent {
         this.setState({
           currentMonth: nextProps.initialVisibleMonth(),
         });
+      } else if (nextProps.visibleMonth) {
+        const newDate = nextProps.visibleMonth();
```

Comment:
Please destructure this out so that nextProps isn’t the receiver of visibleMonth